platform/security/handler/security-handler-pki/pom.xml

```diff
@@ -72,6 +88,7 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
+                            alerts,
```

Comment:
❓ Would adding alerts to [the `platform-alerts` feature](https://github.com/codice/ddf/blob/90c3a3c72ce3c33d8d8979f1104bf32fdce48334/features/admin/src/main/feature/feature.xml#L51) be feasible?